src/Routes/displayQoute.js

```diff
@@ -31,6 +31,7 @@ const Qoutes = () => {
   if (error) {
     return <p>Something went wrong!</p>;
   }
+
   if (loading) return <p>Loading...</p>;
 
   return (
```

Comment:
- [ ] Please fix the `useEffect` hook while fetching the data from the external API, it reloads twice, kindly debug the code, see what's missing, then apply the changes so that when the Quote component loads, it fetches the data only once time, not two!
_Please note that there are no files changed loaded for me to select the right code snippet to highlight._